app/src/test/java/com/codesoom/assignment/domain/UserTest.java

```diff
@@ -31,22 +51,19 @@ void destroy() {
 
     @Test
     void authenticate() {
-        User user = User.builder()
-                .password("test")
-                .build();
+        User user = User.builder().build();
+        user.changePassword("test", passwordEncoder);
 
-        assertThat(user.authenticate("test")).isTrue();
-        assertThat(user.authenticate("xxx")).isFalse();
+        assertThat(user.authenticate("test", passwordEncoder)).isTrue();
+        assertThat(user.authenticate("xxx", passwordEncoder)).isFalse();
     }
 
     @Test
     void authenticateWithDeletedUser() {
-        User user = User.builder()
-                .password("test")
-                .deleted(true)
-                .build();
+        User user = User.builder().deleted(true).build();
+        user.changePassword("test",passwordEncoder);
 
-        assertThat(user.authenticate("test")).isFalse();
-        assertThat(user.authenticate("xxx")).isFalse();
+        assertThat(user.authenticate("test", passwordEncoder)).isFalse();
+        assertThat(user.authenticate("xxx",passwordEncoder)).isFalse();
```

Comment:
별거 아닌 것 같지만 포매터를 사용하거나 `command+option+L`을 자주 눌러줍시다.
```suggestion
        assertThat(user.authenticate("xxx", passwordEncoder)).isFalse();
```

Reply:
자주 까먹네요.. 습관을 키우겠습니다